source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -6866,20 +6870,26 @@ export class Container extends ContainerBase {
             if (typeName) {
                 const registration = context.elementRegistry.findByName(typeName);
                 if (registration?.singletonBehavior !== ElementSingletonBehavior.NotAllowed) {
-                    jsonItems = [jsonItems];
-                    parsingSingletonObject = true;
+                    const element = context.parseElement(
```

Comment:
what is this for?

Reply:
I strengthened the singleton behavior here. The third argument is the forbiddenElements array, which should contain "Carousel" _unless_ the parent container can host singleton elements, and the carousel is the only element in the container.

This behavior will need to change for the updated carousel you're implementing - happy to remove these changes